src/foam/nanos/auth/AgentJunctionStatus.js

```diff
@@ -12,16 +12,22 @@ foam.ENUM({
     {
       name: 'ACTIVE',
       label: 'Active',
+      color: '04612',
```

Comment:
Can you make these use the long form Macros so that they are theme-able?
```
      color: '/*%APPROVAL1%*/ #04612E',
      background: '/*%APPROVAL5%*/ #EEF7ED',
```